src/sentry/testutils/helpers/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .link_header import *  # NOQA
 from .options import *  # NOQA
 from .query import *  # NOQA
+from .redis import *  # NOQA
```

Comment:
ideally we don't want to add things to `__init__.py` and instead explicitly import them

Reply:
Let me revert this